spec/integration/mandatory_user_filter_spec.rb

```diff
@@ -0,0 +1,51 @@
+describe GoodData::MandatoryUserFilter do
+  before :all do
+    @client = ConnectionHelper.create_default_connection
+    @project, * = ProjectHelper.load_full_project_implementation @client
+    @domain = @client.domain(ConnectionHelper::DEFAULT_DOMAIN)
+  end
+
+  after :all do
+    @project.delete if @project
+  end
+
+  describe 'when several users are sharing a MUF' do
+    before :all do
+      label = @project.labels.to_a.first
+      value = label.values.to_a.first
+
+      helper = AppstoreProjectHelper.new(@project, client: @client)
+      usr = helper.ensure_user('tttt@tttt.est', @domain)
+
+      other_usr = @domain.find_user_by_login 'rubydev+admin@gooddata.com'
+
+      filters = [[other_usr.email, label.uri, value[:value]]]
+      @project.add_data_permissions filters
+
+      muf = GoodData::MandatoryUserFilter.all(client: @client, project: @project).to_a.first
+      # TODO: replace this when gooddata-ruby supports MUFs shared between users
+      [usr, other_usr].map(&:uri).each do |usr|
+        payload = {
+          'userFilters' => {
+            'items' => [{
+                          'user' => usr,
+                          'userFilters' => [muf.uri]
+                        }]
```

Comment:
Style/IndentHash: Indent the right brace the same as the start of the line where the left brace is.